RLBotFramework/utils/structures/quick_chats.py

```diff
@@ -119,27 +120,30 @@ def create_queue_for_bot(self, index, team):
         self.bot_queues[index] = (team, bot_queue)
         return queue_holder
 
-    def process_queue(self):
-        while self.game_running:
-            next_message = self.general_chat_queue.get()
-            if next_message is None:
-                time.sleep(0.01)  # sleep for 1/100th of a second
-                continue
-            index, team, team_only, message_details = next_message
-            self.logger.debug('got quick chat from bot %s on team %s with message %s:', index, team,
-                              QuickChats.quick_chat_list[message_details])
-            for i in self.bot_queues:
-                bots = self.bot_queues[i]
-                if i == index:
-                    # do not send yourself a message
-                    continue
-                if bots[0] != team and team_only:
-                    # do not send to other team if team only
-                    continue
-                bots[1].put((index, team, message_details))
-            self.game_interface.send_chat(index, team_only, message_details)
-
-    def start_manager(self):
-        thread = Thread(target=self.process_queue, args=())
+    def process_queue(self, quit_event, quit_callback):
+        while self.game_running and not quit_event.is_set():

```

Comment:
why leave the game running check here if we are using the quit event check?

Reply:
Good point, I'll remove it.